pkgs/tools/virtualization/linode-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, buildPerlPackage, perlPackages, makeWrapper}:
+
+buildPerlPackage rec {
+  name = "linode-cli-${version}";
+  version = "1.4.7";
+
+  src = fetchFromGitHub {
+    owner = "linode";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "1wiz067wgxi4z4rz4n9p7dlvx5z4hkl2nxpfvhikl6dri4m2nkkp";
+  };
+
+  buildInputs = [ makeWrapper ];
+  propagatedBuildInputs = with perlPackages; [
+    JSON
+    LWPUserAgent
+    MozillaCA
+    TryTiny
+    WebServiceLinode
+  ];
+
+  # Wrap perl scripts so they can find libraries
+  postInstall = ''
+    for n in "$out/bin"/*; do
+      wrapProgram "$n" --prefix PERL5LIB : "$PERL5LIB"
+    done
+  '';
+
+  # Has no tests
+  doCheck = false;
+
+  # # Has no docs output
+  # outputs = [ "out" ];
```

Comment:
I think Perl packages are expected to have `devdoc` output, but there is none in this case, which causes the failure.